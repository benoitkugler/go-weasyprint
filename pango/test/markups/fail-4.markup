<span underline_color="#2345">Text</span>
