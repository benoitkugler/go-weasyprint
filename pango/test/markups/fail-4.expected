ERROR: value of 'underline_color' attribute on <span> tag should be a color specification, not '#2345'